src/main/scala/reactive/tweets/incoming/TweetActorManager.scala

```diff
@@ -0,0 +1,25 @@
+package reactive.tweets.incoming
+
+import akka.actor.{Actor, Props}
+import reactive.tweets.domain.{Tweet, User}
+import reactive.tweets.incoming.TweetActor.GetLastTen
+
+class TweetActorManager extends Actor {
+
+  override def receive = {
+    case tweet: Tweet => forward(tweet, tweet.user.name)
+
+    case lastTen: GetLastTen => forward(lastTen, lastTen.user.name)
+
+    case msg => throw new UnsupportedOperationException(s"received unexpected message $msg from ${sender()}")
+  }
+
+  def forward(message: Any, userName: String) = {
+    val tweetPublisherActor = context.child(userName).getOrElse(context.actorOf(TweetActor.props(User(userName)), userName))
```

Comment:
Holy moly, this line is LONG. Why not drop the `val` and add next line's `forward` call to this line as well? :grin: 
Seriously though, one or two extra `val`s wouldn't hurt here IMO.